src/Elastic.Apm/Model/DbSpanCommon.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Data;
+using Elastic.Apm.Api;
+using Elastic.Apm.Helpers;
+
+namespace Elastic.Apm.Model
```

Comment:
This is also not that important, but I'm not sure the `Model` folder is the best place for this. My intention with this folder was that we have the `Api` folder for things that are public and then in the `Model` folder we have model classes that are internal, and those are sent to the server (like `Transaction` and `Span`) - things we don't want to make public but represent our model.

`ExecutionSegmentCommon` is already an exception, and now `DbSpanCommon` would be another one. Maybe we should move those into a separate folder for better code structuring. It's totally fine to do it in a small PR later if we agree.

Reply:
I agree that we can do it in a small PR later.